winlogtimeline/collector/parser.py

```diff
@@ -145,13 +145,13 @@ def parse_id_1074(raw, record):
 
 
 def parse_id_1102(raw, record):
-    data = raw["Event"]["EventData"]["Data"]
+    data = raw["Event"]["UserData"]["LogFileCleared"]
 
     record["description"] = "Log Cleared"
-    record["account"] = data[1]["#text"]
+    record["account"] = data["SubjectUserSid"]
     record["details"] = f'Security event log was cleared by the following account: ' \
```

Comment:
This isn't a change introduced in this PR, but I'm still curious: does interpreting an f-string with no substitutions take longer than if the string weren't an f-string? If so, it may be worthwhile to cut down on any extraneous format string lying around.

Reply:
To follow up, I did some testing on this and found that the differences were negligible.